### PR TITLE
Resize readme image 

### LINKED ISSUE #26: Cryptic README.md - Change grahic
## Discription
Vorschlag: Die Grafik auf cryptic-game/cryptic/Readme.md verkleinern. Den grauen Rand wesentlich kleiner machen, mehr von der eigentlichen Grafik. Wenn einverstanden damit.
Fragen gerne an @DevMimas 

## Design
grafik

### PR DESCRIPTION
**Pull request by cadamini**
**Description**
This pull request resized the images as requested in https://github.com/cryptic-game/cryptic/issues/26

**Issue**
Closes https://github.com/cryptic-game/cryptic/issues/26

**Testing Instructions**
1. Visit https://github.com/cadamini/cryptic/tree/patch-2
2. Checkout the new readme image.

**Additional Notes**
...

### CODE REVIEW

**Comment by DevMimas** [comment]
@cadamini Please align the image left.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-![cryptic](https://raw.githubusercontent.com/cryptic-game/graphics/master/wallpaper/cry_crack.png)
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/1306575/147002513-dfe9cfea-f711-408d-8cfb-ecb6a53e96a8.png" width="400" />
+</p>
```
Comment (cadamini) [inline README.md]:
```suggestion
<img src="https://user-images.githubusercontent.com/1306575/147002513-dfe9cfea-f711-408d-8cfb-ecb6a53e96a8.png" width="400" />
```

**Comment by cadamini** [comment]
Why don't you simply use `https://raw.githubusercontent.com/cryptic-game/cdn/master/images/logo_text.svg`? 

cryptic logo

**Comment by cadamini** [comment]
@DevMimas @Tristan-H11 @Defelo Are there any plans to comment/review/approve this pull request?

**Comment by Tristan-H11** [comment]
Sorry, I did not have this PR in mind.

I'm currently checking if we can use a newer image for the readme. E.g. the one you proposed.

**Comment by DevMimas** [comment]
@cadamini Hey thank you for your help. We will now completely replace the picture with a new, better one.

**Review by DevMimas — APPROVED** [review]

**Review by Defelo — APPROVED** [review]

**Pull request merged**